config/initializers/secure_headers.rb

```diff
@@ -41,9 +41,8 @@
 
     # Add proxy configuration for Angular CLI to csp
     if FrontendAssetHelper.assets_proxied?
-      proxied = ['ws://localhost:3000', 'http://localhost:3000',
-                 'ws://localhost:4200', 'http://localhost:4200',
-                 FrontendAssetHelper.cli_proxy]
+      proxied = ["ws://#{Setting.host_name}", "http://#{Setting.host_name}",
```

Comment:
I think we might need a default value for the `Setting.host_name`, because it can be set to an empty string via the admin interface.
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/83396/208088311-3f6b1f9e-a15a-486f-91f3-3df413c7ac6c.png">

Reply:
Good point.

It already has a [default value](https://github.com/opf/openproject/blob/633b05f495e239d4cfbc6fcf37c065626e61f9eb/config/constants/settings/definitions.rb#L441-L444). Maybe the admin should be changed to forbid empty host name, because it should be set to whatever value is used in the URL to access the application.